### PR TITLE
Local Server Geoprocessing

### PR DESCRIPTION
**Pull request by nCastle1**
This sample demonstrates how to use Local Server to perform geoprocessing tasks. The sample uses a contour geoprocessing service.

### CODE REVIEW

**Comment by nCastle1** [comment]
@dkemlage Can you please give this a review?

File: src/WPF/ArcGISRuntime.WPF.Samples/Samples/LocalServer/LocalServerGeoprocessing/LocalServerGeoprocessing.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+            // set up the sample
+            Initialize();
+        }
+
+        private async void Initialize()
+        {
+            // Create a map and add it to the view
+            MyMapView.Map = new Map(Basemap.CreateLightGrayCanvasVector());
+
+            // Load the tiled layer
```
Comment (dkemlage) [inline src/WPF/ArcGISRuntime.WPF.Samples/Samples/LocalServer/LocalServerGeoprocessing/LocalServerGeoprocessing.xaml.cs]: Perhaps a bit more triple slash comments for the next few lines of code:
 47   +            // Get the file/path to the raster file from the DataManager helper functions  
 48   +            string rasterPath = await GetRasterPath();  
 49   +  
                   // Create a new TileCache from the raster file path
 50   +            TileCache myTileCache = new TileCache(rasterPath);  
 51   +  
                   // Create an ArcGISTiledLater from the TileCache
 52   +            ArcGISTiledLayer tiledLayer = new ArcGISTiledLayer(myTileCache);

File: src/WPF/ArcGISRuntime.WPF.Samples/Samples/LocalServer/LocalServerGeoprocessing/LocalServerGeoprocessing.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+            if (_gpJob.Status == JobStatus.Failed)
+            {
+                MessageBox.Show("Job Failed");
+                return;
+            };
+
+            // Return if not succeeded
+            if (_gpJob.Status != JobStatus.Succeeded) { return; }
+
+            // Get the URL to the map service with the geoprocessing results
```
Comment (dkemlage) [inline src/WPF/ArcGISRuntime.WPF.Samples/Samples/LocalServer/LocalServerGeoprocessing/LocalServerGeoprocessing.xaml.cs]: Perhaps you could break line 163 into two parts and describe what is happening to create the gpServiceResultUrl string. Perhaps you might even
provide a before and after example in the comments of what the gpServiceResultUril would look like.

**Comment by nCastle1** [comment]
Updated from master and made code review changes.

**Pull request merged**